.travis.yml

```
@@ -10,6 +10,4 @@ before_script:
   - pip3 install pre-commit
 script:
   - pre-commit run -a
-# To uncomment when we have meaningful code
-# - npm install --log-level=error
-# - npm run lint
+  - npm run lint
```

Consider using the affected prefix, which will only run on projects and libs that were changed from HEAD master. `npm run lint` will only run linting on the ui project (if you don't specify a project it seems to take the first one)
```suggestion
  - npm run affected:lint
```

Or, if you really want to run linting for all projects, have a look here: https://nx.dev/angular/cli/run-many
Something like this should work:
1. Add a script in the package.json:
```json
"lint-all": "nx run-many --target=lint --all"
```
then you can write the follow in the travis file:
```suggestion
  - npm run lint-all
```

---

Ended up using `nx run-many --all --target=lint` as you mentioned on Slack instead

---

Oh derp I did not read this properly at all LOL, yeah that'll be better and shorter for devs as well. Will add it